pkg/skaffold/deploy/kustomize/kustomize_test.go

```diff
@@ -619,190 +618,6 @@ func TestKustomizeBuildCommandArgs(t *testing.T) {
 	}
 }
 
-func TestKustomizeRender(t *testing.T) {
```

Comment:
Are tests moved to some other place? We don't want to remove these tests

Reply:
Related to kustomize deployer.. I see this https://github.com/ericzzzzzzz/skaffold/blob/main/pkg/skaffold/schema/v2beta28/upgrade.go#L60-L61 from schema upgrade .. if the comment is true should we remove kustomize deployer? However, I do see some breaking changes, like https://github.com/ericzzzzzzz/skaffold/blob/main/pkg/skaffold/schema/v2beta28/upgrade.go#L63-L65 also some fields in this struct
```go
// KustomizeDeploy *beta* uses the `kustomize` CLI to "patch" a deployment for a target environment.
type KustomizeDeploy struct {
	// KustomizePaths is the path to Kustomization files.
	// Defaults to `["."]`.
	KustomizePaths []string `yaml:"paths,omitempty" skaffold:"filepath"`

	// Flags are additional flags passed to `kubectl`.
	Flags KubectlFlags `yaml:"flags,omitempty"`

	// BuildArgs are additional args passed to `kustomize build`.
	BuildArgs []string `yaml:"buildArgs,omitempty"`

	// DefaultNamespace is the default namespace passed to kubectl on deployment if no other override is given.
	DefaultNamespace *string `yaml:"defaultNamespace,omitempty"`

	// LifecycleHooks describes a set of lifecycle hooks that are executed before and after every deploy.
	LifecycleHooks DeployHooks `yaml:"hooks,omitempty"`
}
nil the old kustomize deployer out seems breaking some deployer config and also LifecycleHooks. 
```